app/routes/dive_shop.py

```diff
@@ -0,0 +1,84 @@
+from flask import Blueprint, request
+from app.models import DiveShop
+from app import db
+from flask_jwt_extended import jwt_required, get_current_user
+
+bp = Blueprint('dive_shop', __name__, url_prefix="/dive_shops")
+
+@bp.route('/get', methods=['GET'])
+def fetch_dive_shops():
+  dive_shops = DiveShop.query.all()
+  data = [dive_shop.get_simple_dict() for dive_shop in dive_shops]
```

Comment:
i think this should be the full `get_dict()` instead of get simple dict

Reply:
This is the list of dive shops. We typically keep its items as small as possible and limited to the items we'd conceivably display in a list view.

Reply:
here is an example of where we'd display in a list: https://www.zentacle.com/Beach/64/place-of-refuge-puuhonua-o-hnaunau-national-historical-park
![image](https://user-images.githubusercontent.com/63934597/170054993-92c4bc58-f7ef-4634-af23-946c2c92a65b.png)

In that case we can remove lat/lng, and instead add city and state

Reply:
and we can probably remove owner_id also

Reply:
Done.